Tests/SwerverTests/RequestSpec/HttpRequestSpec.swift

```diff
@@ -7,11 +7,17 @@ class HttpRequestSpec: QuickSpec {
 
     override func spec() {
         describe("HttpRequest") {
-            var httpRequest = HttpRequest(method: RequestMethod.get, url: URL(string: "/")!)
+            guard let url = URL(string: "/") else {
+                return
+            }
+            var httpRequest = HttpRequest(method: RequestMethod.get, url: url)
```

Comment:
What are your thoughts on making the `HttpRequest` constructor private, and introducing an `HttpRequestBuilder`?

#futurepullrequest :)

Reply:
that thought came to me many times but for the sake of not over-engineering, I guess I should leave it as it is until that time comes

Reply:
will also track it in trello